samtranslator/model/iam.py

```diff
@@ -23,7 +23,11 @@ def construct_assume_role_policy_for_service_principal(cls, service_principal):
         document = {
             "Version": "2012-10-17",
             "Statement": [
-                {"Action": ["sts:AssumeRole"], "Effect": "Allow", "Principal": {"Service": [service_principal]},}
+                {
```

Comment:
Just to confirm: You didn't make any change on this file, right?

Reply:
Yes, just formatting. Reverted back